src/Apps/Juno/JsonTypes.hs

```diff
@@ -5,13 +5,16 @@
 module Apps.Juno.JsonTypes where
 
 import           Control.Monad (mzero)
-import           Data.Aeson (genericParseJSON,genericToJSON,parseJSON,toJSON,ToJSON,FromJSON,Value(..))
```

Comment:
Really not a fan of super-long explicit imports AND a qualified import. Just import `Data.Aeson` and `Data.Aeson.Types` unqualified.